src/Discord.Net.WebSocket/Entities/Interaction/Context Menu Commands/Message Commands/SocketApplicationMessageCommandData.cs

```diff
@@ -6,7 +6,7 @@
 namespace Discord.WebSocket
 {
     /// <summary>
-    ///     Represents the data tied with the <see cref="SocketSlashCommand"/> interaction.
+    ///     Represents the data tied with the <see cref="SocketApplicationMessageCommand"/> interaction.
     /// </summary>
     public class SocketApplicationMessageCommandData : SocketEntity<ulong>, IApplicationCommandInteractionData
```

Comment:
Shouldn't this inherit from BaseData? Or is the purpose of that class different?

Reply:
slightly different. one is an interface, the other is a class to support the SocketCommandBase methods. Namely, the Update mthod.